### PR TITLE
System test fixes

### PR DESCRIPTION
**Pull request by willkg**
@milescrabill Do these look ok?

### CODE REVIEW

File: tests/systemtest/test_content_length.py [hunk 1]
```diff
@@ -20,7 +20,8 @@ def http_post(posturl, headers, data):
     if ':' in parsed.netloc:
         host, port = parsed.netloc.split(':')
     else:
-        host, port = parsed.netloc, '80'
+        host = parsed.netloc
+        port = '443' if parsed.startswith('https') else '80'
```
Comment (milescrabill) [inline tests/systemtest/test_content_length.py]: I think you either want `parsed.scheme == https` or `posturl.startswith('https')`.

File: tests/systemtest/test_post_crash.py [hunk 2]
```diff
@@ -221,7 +221,7 @@ def test_regular(self, posturl, s3conn, crash_generator):
         resp = mini_poster.post_crash(posturl, crash_payload, dumps)
 
         # Sleep 1s to give Antenna time to save things
```
Comment (milescrabill) [inline tests/systemtest/test_post_crash.py]: Update the comment too!

File: tests/systemtest/test_post_crash.py [hunk 3]
```diff
@@ -242,7 +242,7 @@ def test_compressed_crash(self, posturl, s3conn, crash_generator):
         resp = mini_poster.post_crash(posturl, crash_payload, compressed=True)
 
         # Sleep 1s to give Antenna time to save things
```
Comment (milescrabill) [inline tests/systemtest/test_post_crash.py]: Update the comment too!

**Comment by willkg** [comment]
^^^ Fixed both issues.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**